lib/oxidized/model/fs.rb

```diff
@@ -0,0 +1,62 @@
+class FS < Oxidized::Model
+  # For switches from FS.com
+  # Author: David Croft <davidc.net>
+  #
+  # tested with:
+  #  - Fiberstore S3900-24F4S
+  #  - Fiberstore S5800-8TF12S
+
+  prompt /^\r?([\w.@()-]+?(\(1-16 chars\))?[#>:]\s?)$/
+  comment '! '
+
+  # handle pager
```

Comment:
Nice work. Just got an FS switch and was looking to integrate it into oxidized :)

I believe the pager handling can be simplified a bit by executing `terminal length 0` which seems to disable the pager:
```
ssh *****************.omniit.dev 
XXXXXXXX@192.168.1.11's password: 

      CLI session with the S3900-24T4S is opened.
      To end the CLI session, enter [Exit].

*****************.omniit.dev-1#terminal length ?
  0         No pausing
  <24-200>  Number of lines on screen
*****************.omniit.dev-1#terminal length 0
*****************.omniit.dev-1#show running-config 
Building running configuration. Please wait...
!<Version>0.0.1</Version>
!<stackingDB>000000000000</stackingDB>
!<stackingMac>0Y_YY-YY-YY-YY-YY-YY_YY</stackingMac>
!<stackingMac>00_00-00-00-00-00-00_00</stackingMac>
!<stackingMac>00_00-00-00-00-00-00_00</stackingMac>
!<stackingMac>00_00-00-00-00-00-00_00</stackingMac>
!<stackingMac>00_00-00-00-00-00-00_00</stackingMac>
!<stackingMac>00_00-00-00-00-00-00_00</stackingMac>
!
....... (no pager :) )
```

Reply:
Yes, actually I added the pager bit before I later added 'term len 0' at login. But I left the pager swallower in place just in case as I don't entirely trust the quality of the engineering of this software, and some models may not support it.